frontend/src/Views/Register.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Avatar,
+  createStyles,
+  CssBaseline,
+  LinearProgress,
+  Paper,
+  Theme,
+  Typography,
+  WithStyles,
+  withStyles,
+} from "@material-ui/core";
+import { PersonAdd } from "@material-ui/icons";
+import React from "react";
+
+import RegistrationForm from "../Components/Register/RegistrationForm";
+
+const styles = (theme: Theme) =>
+  createStyles({
+    main: {
+      width: "auto",
+      display: "block", // Fix IE 11 issue.
+      marginLeft: theme.spacing.unit * 3,
+      marginRight: theme.spacing.unit * 3,
+      [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
+        width: 400,
+        marginLeft: "auto",
+        marginRight: "auto",
+      },
+    },
+    formContainer: {
+      marginTop: theme.spacing.unit * 8,
+    },
+    paper: {
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center",
+      padding: `${theme.spacing.unit * 2}px ${theme.spacing.unit * 3}px ${theme
+        .spacing.unit * 3}px`,
+    },
+    avatar: {
+      margin: theme.spacing.unit,
+      backgroundColor: theme.palette.primary.main,
+    },
+  });
+
+interface IProps extends WithStyles<typeof styles> {}
+
+function Register(props: IProps) {
+  const { classes } = props;
+
+  const [loading, setLoading] = React.useState(false);
+  const [emailError, setEmailError] = React.useState(false);
+
+  const handleRegistration = (email: string, password: string) => {
+    setEmailError(false);
```

Comment:
Gal būtų visai cool turėt tokį on the spot regex email checkerį, kad neišeitų suvest `email` ir taip prisiregistruot
e.g. regex 
```
^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$
```

Reply:
🤔